docs/configuration/version.md

```diff
@@ -265,15 +265,15 @@ it's empty):
 ## Decorating
 
 Decorating phase happens only when version is read (and deserialized).
-During this phase version can be decorated with i.e. branch name.
-Default decorator does nothing. `axion-release-plugin` supports adding
-predefined named version creators (so don't be afraid to post pull
-request if you have something useful!). Decoration phase is conducted by
-*version creators*, you can configure it via `scmVersion.versionCreator`
+During this phase version will be decorated with branch name.
+`axion-release-plugin` supports adding predefined named version creators
+(so don't be afraid to post pull request if you have something useful!).
+Decoration phase is conducted by *version creators*,
+you can configure it via `scmVersion.versionCreator`
```

Comment:
I would state more explicitly that default version creator is `versionWithBranch`

Reply:
check it now :)